ash/src/extensions/nv/device_generated_commands_compute.rs

```diff
@@ -0,0 +1,71 @@
+use crate::vk;
+use crate::{Device, Instance};
+use std::ffi::CStr;
+use std::mem;
+
+/// <https://www.khronos.org/registry/vulkan/specs/1.3-extensions/man/html/VK_NV_device_generated_commands_compute.html>
+#[derive(Clone)]
+pub struct DeviceGeneratedCommandsCompute {
+    handle: vk::Device,
+    fp: vk::NvDeviceGeneratedCommandsComputeFn,
+}
+
+impl DeviceGeneratedCommandsCompute {
+    pub fn new(instance: &Instance, device: &Device) -> Self {
+        let handle = device.handle();
+        let fp = vk::NvDeviceGeneratedCommandsComputeFn::load(|name| unsafe {
+            mem::transmute(instance.get_device_proc_addr(handle, name.as_ptr()))
+        });
+        Self { handle, fp }
+    }
+
+    /// <https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkGetPipelineIndirectMemoryRequirementsNV.html>
```

Comment:
Note that this link 404s. I've reported that upstream. It matches the format used in the function name exactly though.